front/src/applications/editor/tools/rangeEdition/speedSection/EditPSLSection.tsx

```diff
@@ -43,16 +50,53 @@ const getNewRSign = (
   } as PSLSign;
 };
 
+const getNewValidSign = (sign: PSLSign, tracks: TrackSectionEntity[]) => {
```

Comment:
maybe the purpose of the function is more clear like that ?
```suggestion
const validateSignPosition = (sign: PSLSign, tracks: TrackSectionEntity[]): PSLSign => {
```